Prepare for v0.12.0

tiagolobocastro opened the pull request.

tiagolobocastro committed:
chore: prepare for v0.12.0 release

Update py and helm versions
Update CHANGELOG
Update install-guide with breaking changes

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

mhkarimi1383 commented on CHANGELOG.md:
```diff
@@ -22,7 +22,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+- Volumes Snapshots using the rawfile image itself
+  - If the underlying fs supports it, we use `reflinks` for `COW`
+  - Otherwise, a deep copy is performed and application is frozen with `fsfreeze`
```
Here we should mention that we are blocking snapshot create of in use volumes when fsfreeze is not enabled in storage class params

mhkarimi1383 replied:
And users are able to override this behavior by setting params

tiagolobocastro committed:
refactor(helm): update metadata path with release name

Defaults to:
/var/local/openebs/rawfile/{{ .Release.Name }}/meta

This eventually allows for multiple installs, and hardens in case
someone sets up 2 installs without tweaking the default paths.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

tiagolobocastro commented:
@mergify queue
